daml-script/dump/src/main/scala/com/daml/script/dump/Config.scala

```diff
@@ -55,6 +56,14 @@ object Config {
       .text(
         "The transaction offset (inclusive) for the end position of the dump. Optional, by default the dump includes the current end of the ledger."
       )
+    opt[Int]("acs-batch-size")
+      .optional()
+      .action((x, c) => c.copy(acsBatchSize = x))
+      .validate(x =>
+        if (x <= 0) { failure("ACS batch size must be greater than zero") }
```

Comment:
maybe 0 would be a nice way to disable batching?

Reply:
Depends what you mean by disabling batching. If you mean one command per submit, then that's already equivalent to a batch size of 1. If you mean batch all commands into one submit, then yes, we can do that. However, as discussed in https://github.com/digital-asset/daml/pull/8952#discussion_r582953935 that would likely cause issues when more of the ACS contracts are referenced than fit into a tuple. So, I'm not sure if we should provide this option.